src/cli.mjs

```diff
@@ -172,7 +170,6 @@ function buildYargs(argvInput) {
         type: "string",
         description:
           "Profile from the CLI config file to use. Each profile specifies a set of CLI settings.",
```

Comment:
Since there's no longer a separate default column value, but behavior remains the same:
```suggestion
          "Profile from the CLI config file to use. Each profile specifies a set of CLI settings. Defaults to the 'default' profile.",
```